### PR TITLE
Fix exception when attempt to load image from file system

### PR DESCRIPTION
**Pull request by harry-nguyen-88**
## Summary

When trying to load image using method "RCTImageFromLocalAssetURL", if URL's path in file system representation return null (in my case, imageUrl is base64 format returned from server so it's always null) then method "stringWithUTF8String" will raise an exception.

## Changelog

[iOS] [Fixed] - Fix exception when attempt to load image from file system

## Test Plan

If we have base64 image with imageURL is something like "data:image/jpeg;base64,/9j/4AAQSkZJRgABAQAASABIAAD/4QBYRXhpZg...." and using RCTImageFromLocalAssetURL to load it, we can load the correct image without any exception.

### CODE REVIEW

File: React/Base/RCTUtils.m [hunk 1]
```diff
@@ -740,11 +740,14 @@ BOOL RCTIsLocalAssetURL(NSURL *__nullable imageURL)
 
   if (!image) {
     // Attempt to load from the file system
-    NSString *filePath = [NSString stringWithUTF8String:[imageURL fileSystemRepresentation]];
-    if (filePath.pathExtension.length == 0) {
-      filePath = [filePath stringByAppendingPathExtension:@"png"];
+    const char* fileSystemCString = [imageURL fileSystemRepresentation];
+    if (fileSystemCString != nil) {
```
Comment (shergin) [inline React/Base/RCTUtils.m]: It's not idiomatic to compare raw C pointer to nil, we should use null instead or rely on implicit conversion to bool.

File: React/Base/RCTUtils.m [hunk 2]
```diff
@@ -740,11 +740,14 @@ BOOL RCTIsLocalAssetURL(NSURL *__nullable imageURL)
...
+      NSString *filePath = [NSString stringWithUTF8String:[imageURL fileSystemRepresentation]];
```
Comment (shergin) [inline React/Base/RCTUtils.m]: Why not use `fileSystemCString` here?

File: React/Base/RCTUtils.m (hunk 1 above)
Comment (harry-nguyen-88) [reply React/Base/RCTUtils.m]: Agreed! I will fix it!

File: React/Base/RCTUtils.m (hunk 2 above)
Comment (harry-nguyen-88) [reply React/Base/RCTUtils.m]: My bad! I forgot to review the code there.

**Pull request closed without merging**